package.json

```diff
@@ -90,6 +90,7 @@
     "@types/uuid": "^8.0.0",
     "binary-search-bounds": "^2.0.4",
     "c8": "^7.0.0",
+    "chai": "^4.2.0",
```

Comment:
If this is only needed for the sample tests, it shouldn't be here, and instead should be in `devDependencies` of `package.json` in `samples/package.json`

Reply:
@JustinBeckwith good point! I see that `chai` is already in the dev dependencies of the samples `package.json`. I am, however, getting an error on running the sample libraries as per the `CONTRIBUTING.md` file:
```
npm run samples-test
```
Maybe we are missing a step on the `.md` instead?

Reply:
Yeah, that's the gap here 🤦   You can run the samples tests in 1 of 2 ways:
```
npm install
cd samples
npm install
cd ..
npm run samples-test
```

Or

```
npm install
cd samples
npm install
npm test
```

The `CONTRIBUTING.md` is centrally managed, so we'll need to cut a PR out to all 94 repos for this one.  Apologies for the confusion!

Reply:
No worries, glad we identified the problem! I will close this PR. Let me know if you need any help with the PR in the centralized repo.

Cheers,